sherpa/bin/pruned_stateless_emformer_rnnt2/decode.py

```diff
@@ -141,8 +148,24 @@ def __init__(
         self.num_fetched_frames = 0
 
         self.states = initial_states
-        self.decoder_out = decoder_out
+        self.decoding_graph = decoding_graph
+
+        if decoding_method == "fast_beam_search":
+            assert decoding_graph is not None
+            self.rnnt_decoding_stream = k2.create_rnnt_decoding_stream(
+                decoding_graph.arcs
+            )
+            self.hyp = []
+        elif decoding_method == "greedy_search":
+            assert decoder_out is not None
+            self.decoder_out = decoder_out
+            self.hyp = [blank_id] * context_size
+        else:
+            assert (
```

Comment:
`assert False` throws an `AssertionError` exception.
I suggest
```python3
raise ValueError(f''Decoding method {decoding_method} is not supported.")
```
It expresses the intention directly.

Reply:
Done